.github/workflows/ci.yml

```diff
@@ -42,10 +44,6 @@ jobs:
       - name: Install NPM dependencies
         run: npm install
 
-      # delete me after updating purescript-contrib/setup-purescript@main
-      - name: Install latest spago
-        run: npm install -g spago@next
```

Comment:
Honestly we could just ditch setup-purescript here and install purs and spago via npm so the libraries aren’t relying on each other.

Reply:
ok, fixed

This workflow is awaiting approval from a maintainer in https://github.com/purescript-contrib/purescript-github-actions-toolkit/pull/20